src/drafts/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -403,7 +403,8 @@ const MarkdownEditor = forwardRef<MarkdownEditorHandle, MarkdownEditorProps>(
                 '&: focus-within':
                   view === 'edit'
                     ? {
-                        outline: '2px solid var(--borderColor-accent-emphasis)',
+                        outline: '2px solid',
+                        outlineColor: 'accent.emphasis',
                       }
                     : {},
```

Comment:
Should we capture this in a new issue? It seems like Firefox will support `:has` by Christmas: https://groups.google.com/a/mozilla.org/g/dev-platform/c/oacuvZ2_hLg/m/4o28pFLkAwAJ?pli=1 🎉

Reply:
Oh that's interesting. We already use `:has` in [primer/react in a few places](https://github.com/search?q=repo%3Aprimer%2Freact+%3Ahas&type=code). I'm assuming this is because dotcom uses a polyfill